packages/framework/synthesize/src/IFluidDependencySynthesizer.ts

```diff
@@ -61,22 +30,15 @@ export interface IFluidDependencySynthesizer extends IProvideFluidDependencySynt
      * @param requiredTypes - required types that need to be in the Scope object
      */
     synthesize<
-        O extends IFluidObject,
-        R extends IFluidObject>(
+        O,
+        R = undefined  | Record<string, never>>(
```

Comment:
I'm not sure it's desirable to permit `undefined` here, vs. just the never record?  I'd probably prefer the stricter option to start at least.

Reply:
i don't actually want the never record, but that's what we supported before. so there are existing usages that pass and emtpy object, it's better to just pass undefined, so you don't need to create and object.